frontend/src/App.css

```
@@ -13,6 +13,10 @@
   }
 }
 
+body {
+  background-color: #aecbc9; /* Replace with the color you want */
```

```suggestion
  background-color: #aecbc9;
```
assuming this is the color you want!